.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This line might be causing issues

Reply:
I can remove it, but the same line causes no issues in [this](https://github.com/exercism/generic-test-runner/blob/master/.github/workflows/main.yml) workflow.

Reply:
Also, shouldn't Actions be running, but failing due to a workflow config parse failure?